### PR TITLE
Added more documentation for merging with PRs with ASF master.

### PR DESCRIPTION
**Pull request by prestoncarman**

### CODE REVIEW

File: src/site/apt/development_contribution.apt [hunk 1]
```diff
@@ -96,11 +100,25 @@ Contributing Code
...
+
+      Update your change to address any comments from reviewers.
+
+    * Prepare your change for merge.
+
+      Squash your changes into a single commit with a nice commit message.
+      The commit message's first line should be less than 50 character and any additional comments
+      are included below a blank line.
+
+      Git rebase has a option of merging commits into a single commit that works nicely for squashing your changes.
```
Comment (westmann) [inline src/site/apt/development_contribution.apt:113]: Could we add an example here?

File: src/site/apt/development_contribution.apt [hunk 2]
```diff
@@ -114,9 +132,25 @@ Contributing Code
...
   Often the committer will also be the reviewer for non-committer changes.
 
+    * Add ASF as a git remote (first time committers).
+
+      Create a git remote for ASF repository. {{{https://git-wip-us.apache.org/repos/asf/vxquery.git}}}
+
     * Double check the VXQuery and XQTS tests.
 
+      A {{{https://asterix-jenkins.ics.uci.edu/job/vxquery-pr/}VXQuery Jenkins}} instance has been set up to
+      check the last ten Pull Requests.
```
Comment (westmann) [inline src/site/apt/development_contribution.apt:143]: What are we checking there? Should we add that to the docs or does it get outdated too fast?

File: src/site/apt/development_contribution.apt [hunk 3]
```diff
@@ -114,9 +132,25 @@ Contributing Code
...
+
+    * Double check the change.
+
+      Confirm the change has a single commit and includes a nice commit message.
+
     * Merge the change with ASF master.
 
-      When merging the change, do not rebase.
+      When merging the change, do not ~rebase~.
```
Comment (westmann) [inline src/site/apt/development_contribution.apt:154]: Example?

**Comment by westmann** [comment]
These instructions are nice, but I think that they'd be even better with example command lines.

**Comment by prestoncarman** [comment]
Merged change into the site prep for the next release.

**Pull request closed without merging**